src/build123d/build_line.py

```diff
@@ -748,28 +734,26 @@ class Spline(Edge):
     def __init__(
         self,
         *pts: VectorLike,
-        tangents: Iterable[VectorLike] = None,
-        tangent_scalars: Iterable[float] = None,
+        tangents: Collection[VectorLike] = None,
```

Comment:
I'm not a typing expert but isn't `Collection` more restrictive than `Iterable` ?

Reply:
Yes indeed Collection is more restrictive in that it provides a __len__() dunder method that is used a few lines below.